targets/circuitplay-express.json

```diff
@@ -0,0 +1,5 @@
+{
+    "inherits": ["atsamd21g18a"],
+    "build-tags": ["sam", "atsamd21g18a", "circuitplay_express"],
```

Comment:
The build tags `sam` and `atsamd21g18a` are already defined in atsamd21g18a.json, so they do not need to be redefined here.

Reply:
OK will remove.

Reply:
If I remove them from here, making it

```json
"build-tags": ["circuitplay_express"],
```

then when building I get the error:

```shell
error: no buildable Go source files in /home/ron/.gvm/pkgsets/go1.11/global/src/github.com/tinygo-org/tinygo/src/device/sam
```

Reply:
I see why: atsamd21g18a.json does not set the `atsamd21g18a` build tag.

Reply:
I had a feeling i was not using it quite correctly. OK will fix in separate PR.